runtime/src/main/java/com/redhat/ipaas/rest/Users.java

```diff
@@ -37,12 +40,16 @@
     @Inject
     private DataManager dataMgr;
 
+    @Context
```

Comment:
yeah, thats probably better as its more direct. tbh, I always have to think how this is injected on a per request basis. Always like to think of endpoints as immutable singletons, but thats not the reality. 

The original motivation here was to keep method signature clean, not cluttering it with 'technical' details. 

will change it.